lib/brain.js

```diff
@@ -2101,14 +2158,24 @@ Brain.prototype.setManualTrigger = function setManualTrigger (code) {
   }
 }
 
-Brain.prototype.isTierCompliant = function isTierCompliant (tier, triggerTx, requiredTiers) {
+Brain.prototype.isTierCompliant = function isTierCompliant (trigger, triggerTx, requiredTiers, _tier) {
   const tx = this.tx
   const customer = this.customer || {}
 
-  // custom requests have uuid
-  const tierOrCustom = uuid4.validate(tier) ? 'custom' : tier
```

Comment:
Not anymore, UUID was used in a really hacky way to determine if a trigger was a custom info request or not. Basically, if the trigger.tier was a UUID and not a string like `sms` or `usssn`, it would just consider it as a 'custom' tier trigger. We can just use the fields that actually describe a custom info request trigger, instead of this hacky way that was begging for unintended behavior to happen